src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -847,6 +847,10 @@ def load_arguments(self, _):
                        help='The nodepool id.', validator=validate_nodepool_id)
             c.argument('aks_custom_headers')
 
+    with self.argument_context('aks nodepool snapshot update') as c:
+        c.argument('snapshot_name', options_list=['--name', '-n'], required=True, help='The nodepool snapshot name.', validator=validate_snapshot_name)
+        c.argument('tags', tags_type, required=True, help='The tags to set to the snapshot.')
```

Comment:
```suggestion
        c.argument('snapshot_name', options_list=['--name', '-n'], help='The nodepool snapshot name.', validator=validate_snapshot_name)
        c.argument('tags', tags_type, help='The tags to set to the snapshot.')
```
Same as https://github.com/Azure/azure-cli/pull/26790/files#r1253829879